controllers/gslb_controller_test.go

```diff
@@ -650,7 +651,7 @@ func TestGslbController(t *testing.T) {
 			t.Fatalf("Can't setup env var: (%v)", err)
 		}
 		resolver := depresolver.NewDependencyResolver(context.TODO(), cl)
-		r.config, err = resolver.ResolveOperatorConfig()
+		r.Config, err = resolver.ResolveOperatorConfig()
```

Comment:
I created [an additional issue](https://github.com/AbsaOSS/k8gb/issues/168) for that